manifests/rhoai/shared/apps/model-catalog/model-catalog-configmap.yaml

```diff
@@ -0,0 +1,366 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: model-catalog-source-redhat
+  labels:
+    opendatahub.io/dashboard: "true"
+data:
+  modelCatalogSource: |
+{
```

Comment:
@alexcreasy testing this, I get a parsing error trying to save it with the current spacing. I think the JSON needs to be indented past the `modelCatalogSource: |` line like:

```yaml
data:
  modelCatalogSource: |
    {
      "source": "Red Hat",
```
When I change it to this and save the configmap I get no error.